mafiasi/gprot/views.py

```diff
@@ -455,3 +458,27 @@ def delete_reminder(request, reminder_pk):
     reminder.delete()
 
     return redirect('gprot_reminders')
+
+@login_required
+def favorite(request):
+    action = request.POST.get('action', None)
+    user = request.user
+    url = request.POST.get('url', None)
```

Comment:
Same as above, and what happens if `url` is `None`? `re.split` below will throw a TypeError. So you should treat the possibility.